### PR TITLE
Fix for timeslider solr param not being encoded properly

### PR DESCRIPTION
**Pull request by jesperlauridsen**
Readded label function that went missing (??!?)
Fixed encoding problem after tomcat upgrade
Fixed minor issue when reloading the page after implementing the sort option.

### CODE REVIEW

File: src/components/AppliedFilters.vue [hunk 1]
```diff
@@ -99,12 +99,27 @@ export default {
     },
     findCategory(filter) {
       let category;
-      if (filter.includes("&pt=")) {
+      if (filter.includes("&pt=") === true) {
```
Comment (jorntx) [inline src/components/AppliedFilters.vue:102]: Why?

File: src/containers/SearchContainer.vue [hunk 2]
```diff
@@ -71,8 +74,19 @@ export default {
   },
 
   beforeRouteEnter(to, from, next) {
-    const query = to.params.query;
+    //Apparently full encoding breaks solr, so we just want to escape the brackets [], since that's what needed for the tomcat to accept the query.
+    const query = to.params.query.replace(/\[/g,'%5B').replace(/]/g,'%5D');
```
Comment (jorntx) [inline src/containers/SearchContainer.vue:78]: I see your point

File: src/components/AppliedFilters.vue (hunk 1 above)
Comment (jesperlauridsen) [reply src/components/AppliedFilters.vue:102]: Nooooo! I thought i had fixed that!

Comment (jesperlauridsen) [reply src/components/AppliedFilters.vue]: Now it's gone!

**Pull request merged**